### ISSUE TITLE
Unable to run openslides-portable on Win10

### ISSUE THREAD

**Issue by Hackwar** [OpenSlides/openslides-portable#38]
I'm testing OpenSlides again and the portable version works fine on my Win7 system. Now a friend of mine downloaded the portable version and can't run it. Starting the openslides.exe (even with admin rights) results in an error "failed to load python37.dll error 126"

Installing python 3.7 on the system makes that error go away, instead it simply says "Could not execute openslides" and then it closes itself again.

The portable package was unzipped onto the desktop and also in the documents folder, in the hopes that it might resolve permission issues. But no luck

I tried to troubleshoot the issue with Teamviewer, but could not find the reason for the problem. Any ideas?

**Comment by emanuelschuetze** [comment]
Which version you tested? OpenSlides 2.3 portable?
Which Windows version use your frind?
For more debug you can try to run `openslides_console.exe` from windows cmd. This should get more debug output why openslides gui is not startet.

**Comment by emanuelschuetze** [comment]
Which version you tested? OpenSlides 2.3 portable?
Which Windows version use your frind?
For more debug you can try to run `openslides_console.exe` from windows cmd. This should get more debug output why openslides gui is not startet.

**Comment by Hackwar** [comment]
Yes, we tested the 2.3 version that you can download from the OpenSlides website. I don't know which Win10 version it is exactly. I'll see if I can provide more information. We'll try to check it on cmd.

**Comment by Hackwar** [comment]
Ok, we finally got a minute to test this out and openslides_console.exe returned an issue with wx:
```
  File "C:\Program Files\Python\Python37\lib\site-packages\wx\__init__.py", line 17, in <module>
    from wx.core import *
  File "C:\Program Files\Python\Python37\lib\site-packages\wx\core.py", line 12, in <module>
    from ._core import *
ImportError: DLL load failed: Module not found.
```
(This is a similar message. I actually forgot to copy the real message from her terminal over teamviewer.)

After some googling, I came to the conclusion that I had to install the "Visual C++ Redistributable for Visual Studio 2015" and thus did that. That did NOT solve the issue, because I got the original release, but that contained a bug that actually created this issue. Long story short, you need "Visual C++ Redistributable for Visual Studio 2015 ***Update 3***" to fix the issue. Installing that solved my issue. I don't know if you can introduce a check for this lib and give a more meaningfull error message?
